packages/css/package.json

```diff
@@ -4,13 +4,13 @@
     "url": "https://www.sick.com"
   },
   "dependencies": {
-    "@sick-design-system/design-tokens": "workspace:^",
-    "@sick-design-system/icons": "workspace:^"
+    "@synergy-design-system/design-tokens": "workspace:^",
+    "@synergy-design-system/icons": "workspace:^"
   },
   "description": "Various CSS styles used in the SICK Design System",
```

Comment:
Do we also need to adapt the naming here and say "Synergy Design System" instead of "SICK Design System"? Same question for the other package.json files

Reply:
Good catch, will update accordingly

Reply:
Done in latest push